ui/src/scss/_patterns_icons.scss

```diff
@@ -113,6 +113,24 @@
     background-size: 100% 100%;
   }
 
+
+  .p-icon--debug {
```

Comment:
Vanilla got a bunch of new icons recently. Do you think any of them could work here instead?

Reply:
Yep, I'll use the debug one, it looks a bit different, but probably better than a custom icon.